addons/account/migrations/9.0.1.1/tests/test_account.py

```diff
@@ -1,13 +0,0 @@
-# coding: utf-8
```

Comment:
I thought about moving the test. The problem is: account_full_reconciliation is not really migrated, because the module did not exist before. Then maybe the OpenUpgrade code should be changed to run migrations based on wether or not there is a migrations directory in a module, even for modules that did not exist in the previous version. As yet had no time to investigate this.